.github/workflows/main.yml

```diff
@@ -3,69 +3,75 @@ on:
   push:
     branches:
       - master
-  pull_request:
+      - feature/new_node_setup
   schedule:
-    - cron: "0 2 * * 0"
+    - cron: 0 2 * * 0
+permissions:
```

Comment:
The only required things here are probably contents (RO) and packages (RW). All other permissions will be automatically disabled.

Reply:
Needs proper review of permissions, it's a checkbox for now as there is no risk of having too much. It's still tighter than it was weeks ago.